### DISCUSSION TITLE
Database usage

### DISCUSSION THREAD

**Discussion by mashrurbd** [supabase/supabase#9758] [Questions]
Please clarify this: 

I have a table in the database and its only 32KB in size. 

So why it is showing a Database Space Usage under Settings > Database page as 30 MB out of 512 MB?

Does this resets every month?

**Comment by mashrurbd** [comment]
@SupaDavid

**Reply by SupaDavid** [reply]
Hi @mashrurbd , if you could raise a support ticket for this via https://app.supabase.io/support/new , our team should be able to investigate for you
